### PR TITLE
Standardize text prompting to connect/authenticate the user to wordpress.com

### PR DESCRIPTION
**Pull request by dereksmart**
Fixes inconsistent messaging to authenticate the user with WordPress.com

#### Changes proposed in this Pull Request:

Updates all copy to use the same messaging for connecting/authenticating the user with WordPress.com.

Before - Dashboard: 
image

After - Dashboard: 
image

Before - Stats dashboard button:
image

After - Stats dashboard button:
image

Before - Settings card bar:
(This is just one - there are many settings bars that change in the PR, please look at the code changes)
image

After - Settings card bar: 
image

#### Jetpack product discussion

p1HpG7-bRQ-p2#comment-46172

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

Set up Jetpack in site-only connection mode, and view the respective areas. Check for:
- Spacing
- Typos
- Wording

### CODE REVIEW

**Comment by folletto** [comment]
Since we are here, can we do these tweaks?

**Account connection**
Remove the duplicate, changing to a single sentence with the first part linked.

> Connect your WordPress.com account to get the most out of Jetpack

**Settings card bar**
Change the button to not wrap text, this could be as simple as "white-space: nowrap;".

As for the text, I'd tweak the left side:

> This feature is provided by the WordPress.com cloud. Connect to set up your status alerts.

Given the action is already the button, no need to repeat.

File: projects/plugins/jetpack/_inc/client/sharing/publicize.jsx [hunk 1]
```diff
@@ -109,7 +109,10 @@ export const Publicize = withModuleSettingsFormHelpers(
 						<ConnectUserBar
 							feature="publicize"
 							featureLabel={ __( 'Publicize', 'jetpack' ) }
-							text={ __( 'Sign in to connect your social media accounts.', 'jetpack' ) }
+							text={ __(
+								'Connect your WordPress.com account to connect your social media accounts.',
```
Comment (keoshi) [inline projects/plugins/jetpack/_inc/client/sharing/publicize.jsx:113]: This is the only one that raises some concerns to me — would suggested either of these with a slight preference for the first:

- Connect your WordPress.com account to manage your social media accounts.
- Connect your WordPress.com account to add your social media accounts.

**Review by keoshi — CHANGES_REQUESTED** [review]
Thanks for doing this so quickly, @dereksmart ! This is _churrific_.

Only had one comment, everything else looks great.

**Comment by dereksmart** [comment]
@keoshi @folletto great suggestions, thank you. I've implemented all suggestions and updated the screenshots in the OP. Settings cards look much better! 

I've taken this opportunity to match the disconnect user text (only shown to secondary connections) to match the connection text. See that change in ae9fcd3

Erin - the Account Connection card is a bit tricky - that link text is technically a button component, and would take a little bit of refactoring that I don't have time for right now. In the meantime, what do you think of this? 

image

**Comment by folletto** [comment]
> Erin - the Account Connection card is a bit tricky - that link text is technically a button component, and would take a little bit of refactoring that I don't have time for right now. In the meantime, what do you think of this?

Ah! Thanks for the clarification there.

I think as a quick fix that works, thank you!

File: projects/plugins/jetpack/_inc/client/sharing/publicize.jsx (hunk 1 above)
Comment (dereksmart) [reply projects/plugins/jetpack/_inc/client/sharing/publicize.jsx]: Done in 86f848d

**Review by jeherve — APPROVED** [review]
Looks good to me.

**Comment by jeherve** [comment]
Cherry-picked to `jetpack/branch-9.7` in a9cb8b53030783a0bf3bda04625433620f3121dc

**Pull request merged**